.travis.yml

```diff
@@ -22,7 +22,7 @@ install:
     - composer install
 
 script:
-    - phpunit --coverage-text
+    - vendor/phpunit/phpunit/phpunit --coverage-text
```

Comment:
use ``vendor/bin/phpunit`` instead of relying on the path inside phpunit itself (which is wrong btw, this is *not* the file exposed as a composer bin in all versions)